cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -281,16 +281,14 @@ def __str__(self):
         msg_lines = str(self.circuit).split('\n')
         msg_width = max([len(line) for line in msg_lines])
         circuit_msg = '\n'.join(
-            '[ {line:<{width}} ]'.format(line=line, width=msg_width) for line in msg_lines
+            '{line:<{width}}'.format(line=line, width=msg_width) for line in msg_lines
         )
         args = []
 
         def dict_str(d: Dict) -> str:
             pairs = [f'{k}: {v}' for k, v in sorted(d.items())]
             return '{' + ', '.join(pairs) + '}'
 
-        if self.qubit_map:
-            args.append(f'qubit_map={dict_str(self.qubit_map)}')
```

Comment:
I don't think I support removal of `qubit_map`, even if in many cases the mapping is apparent from how the subcircuit and its component qubits are drawn in the larger circuit.

For example, consider this case:
```
q0, q1, q2, q3 = cirq.LineQubit.range(4)
op = cirq.CircuitOperation(cirq.FrozenCircuit(cirq.CNOT(q3, q2)))
c = cirq.Circuit(op.with_qubit_mapping({q2: q3, q3: q2}))
print(c)
# output:
# 2: ───#2───────────────
#       │
#       ┌────────────┐
#       │ 2: ───X─── │
# 3: ───│       │    │───
#       │ 3: ───@─── │
#       └────────────┘
```
This is equivalent to `cirq.CNOT(q2, q3)`, but without the explicit mapping it would be very easy to misread as `cirq.CNOT(q3, q2)`.